infrastructure/metadata/infrastructure/testdriver/actions/actionsWithKeyPressed.html.ini

```diff
@@ -6,4 +6,5 @@
   [TestDriver actions: actions with key pressed]
     expected:
       if product == "firefox": FAIL
+      if os == "mac": FAIL
```

Comment:
Should this be specific to chrome? If not any reason it fails for all browsers on mac?

Reply:
It fails on mac for chrome .